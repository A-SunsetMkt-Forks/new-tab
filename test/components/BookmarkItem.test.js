'use strict';

const BookmarkItem = require('../../src/components/BookmarkItem.html');

const _node = {};

describe('BookmarkItem component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new BookmarkItem({
      target,
      data: {
        _node,
      },
    });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
