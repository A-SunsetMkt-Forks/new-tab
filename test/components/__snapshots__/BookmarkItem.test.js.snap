// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkItem component renders correctly 1`] = `
"<a href=\\"undefined\\" title=\\"undefined\\" class=\\"bookmark\\">
    <span class=\\"title\\">undefined</span></a><!---->"
`;
